scripts/lib/stringify-and-order-properties.ts

```diff
@@ -36,7 +36,7 @@ const propOrder = {
       'description',
       'mdn_url',
       'spec_url',
-      'matches',
```

Comment:
Good catch: `matches` don't exist (anymore).

Note that `source-file` exists in the schema but is not listed. Should we add it?